pom.xml

```
@@ -99,7 +99,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git-client</artifactId>
-      <version>3.0.0-beta5</version>
+      <version>3.0.0-beta6-SNAPSHOT</version>
```

You'll need to use an incrementals version here, not a snapshot.
```suggestion
      <version>3.0.0-beta6-rc1822.65df57d49890</version>
```